### DISCUSSION TITLE
What is the JWT Token?

### DISCUSSION THREAD

**Discussion by GentNiginz** [Zibbp/ganymede#106] [General]
Hello everyone,

I'm currently trying to understand the composer file and don't quit understand the use of JWT tokens and how I can get them...

I'd greatly appreciate any help,

Thanks!

**Comment by Zibbp** [comment]
Hi,
The JWT Secrets in the `docker-compose.yml` file are used when signing the JWT token. JWT tokens are tokens that have data inside used for authenticating the user. The secret is used to sign and verify the token was signed by that secret.

For the JWT secret and JWT refresh secret, they need to be set to random strings. Generate a random string how you see fit and use that.

**Reply by GentNiginz** [reply]
Oh thank you very much!
